CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `RedshiftSpectrum` source to the library.
 - Added `upload()` and `download()` methods to `S3` source.
 - Added `Genesys` source to library.
+- Fixed bug in `Databricks._full_refresh()`.The function replacing column names with snake_case was not used. (#672)
```

Comment:
"fixed bug" -> "fixed a bug"; missing space after dot. You can use eg. Code Spell Checker VSCode extension to avoid doing these kind of mistakes in the future

Reply:
[📝 Improved entry in CHANGELOG.md](https://github.com/dyvenia/viadot/pull/673/commits/a78bc7ec3f216ba8efd6f9b3a30708eaa6146b2f)